### ISSUE TITLE
Not detect string interpolation - babel-plugin-transform-react-pug

### ISSUE THREAD

**Issue by PhmNgocNghia** [ezhlobo/eslint-plugin-react-pug#26]
***string interpolation**
Error : react-pug/no-broken-template
Code that got error: [https://paste.ofcode.org/8SRSabwhEnQBwYeTNdEXDG](code)
Transpilier use: https://github.com/pugjs/babel-plugin-transform-react-pug
Code that cause error:
`div(className=${portfolioSection.projects > 0 ? '.mb-4' : ''})`

**Comment by ezhlobo** [comment]
@PhmNgocNghia thanks for reporting this. I was able to reproduce this behavior, will keep you updated.

**Comment by ezhlobo** [comment]
@PhmNgocNghia in the meantime you can see the better options of using pug in react: https://github.com/pugjs/babel-plugin-transform-react-pug#define-local-variables-and-use-javascript-in-attributes

Basically your example could be re-written this way:

```pug
div(class=portfolioSection.projects > 0 && '.mb-4')

div(class=portfolioSection.projects > 0 ? '.mb-4' : '')
```

**Comment by ezhlobo** [comment]
@PhmNgocNghia I've published new release: `0.3.2`. Please, take a look at it and feel free to re-open this issue if it's not fixed.
